api/handler.py

```diff
@@ -0,0 +1,70 @@
+import json
+import os
+from api import get_co2_estimate
+from transport_co2 import Mode
+
+def strtofloat(val):
+    if isinstance(val, str):
+        val = float(val)
+    return val
+
+def post(event, context):
+    try:
+        vars = event.get("queryStringParameters", dict())
+        if vars != None:
```

Comment:
I'd say it's cleaner to do this:

```python
if not vars:
            response = {
                "statusCode": 400,
                "body": json.dumps({
                    "error:": "No parameters provided",
                })
            }
            return response
```

and then have the rest unindented by one level.

Reply:
done